recipes/pynumero_build/meta.yaml

```diff
@@ -0,0 +1,33 @@
+{% set name = "pynumero" %}
+{% set version = "0.0.0" %}
+{% set variant = "openblas" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_rev : v0.0.0
```

Comment:
Is it possible to build from a tarball instead of the git repo?

https://conda-forge.org/docs/meta.html#build-from-tarballs-not-repos

Reply:
Sure. I will try to getting working with a tarball